var/spack/repos/builtin/packages/py-versioneer/package.py

```diff
@@ -10,12 +10,14 @@ class PyVersioneer(PythonPackage):
     """Versioneer is a tool to automatically update version strings by
     asking your version-control system about the current tree."""
 
-    homepage = "https://github.com/warner/python-versioneer"
-    url = "https://github.com/warner/python-versioneer/archive/0.18.tar.gz"
-    git = "https://github.com/warner/python-versioneer.git"
+    homepage = "https://github.com/python-versioneer/python-versioneer"
+    pypi = "versioneer/versioneer-0.26.tar.gz"
+    git = "https://github.com/python-versioneer/python-versioneer.git"
 
     maintainers = ["scemama"]
 
+    version("0.26", sha256="84fc729aa296d1d26645a8f62f178019885ff6f9a1073b29a4a228270ac5257b")
```

Comment:
Confirmed sha256 for 0.26 BUT the URL change means the sha256 for v 0.18 is no longer valid.

Easiest thing to do would be to add the URL to v0.18.  Or you can change the sha256 for 0.18 to `ead1f78168150011189521b479d3a0dd2f55c94f5b07747b484fd693c3fbf335`

Reply:
Yes, you are right, I forgot about that. This is fixed now.
Thanks!